### ISSUE TITLE
Aiaccel not working on Windows 11

### ISSUE THREAD

**Issue by styamamoto** [aistairc/aiaccel#164]
On Windows 11, aiaccel stopped due to a TypeError.

**To Reproduce**
1. Go to 'Examples/sphere'
2. Execute `aiaccel-start --config config.yaml --celean` or `python -m aiaccel.cli.start --config config.yaml --clean`
3. Then, TypeError arose.

**Desktop**
 - OS: Windows 11
 - Python 3.8.10, 3.10.9

aiaccel.util.process.ps2joblist() の 91 行目で `p_info.info['cmdline'])` が `None` を返す場合があり，' '.join() しようとしたところでエラーが発生しました．
https://github.com/aistairc/aiaccel/blob/d297a5203b973b299ced5c4b2a95756702721670/aiaccel/util/process.py#L73-L97

**Comment by tkmnet** [comment]
In my opinion, native Windows does not have to be supported, because WSL has become popular.
If native Windows have to be supported, the implementation able to replace with [Psutil](https://pypi.org/project/psutil/) library.

**Comment by 3bisuoka** [comment]
Trying to use aiaccel on my Intel Mac, I just saw the same error with Psutil library.

**Comment by tkmnet** [comment]
I am sorry. I missed the implementation of using Psutil.

I think security reasons cause it. Therefore, I think it should be given default values.
```
p_info.info.get('cmdline', '<not available>')
```

**Comment by styamamoto** [comment]
I think such the replacement seems not to work well because the key 'cmdline' exists (but the corresponiding value is `None`).

**Comment by tkmnet** [comment]
I see... but the important point is that there is a possibility that it can be replaced with a dummy string.

**Comment by styamamoto** [comment]
To resolve this, pull request is created. https://github.com/aistairc/aiaccel/pull/172
In the pull request, following is employed;
```
" ".join(p_info.info['cmdline'] or [])
```

**Comment by styamamoto** [comment]
The problem discussed above was successfully resolved by PR #172.
However, I found another problem now;
Optimization had started but stopped by `ModuleNotFoundError` from user.py.
(_e.g._ line 1 of aiaccel/examples/sphere/user.py: `import numpy as np`)
This exception occured when using `local` mode, but `python_local` mode worked well without such the exception.
(I did not check `ABCI` mode)

**Comment by styamamoto** [comment]
> Optimization had started but stopped by ModuleNotFoundError from user.py.

To resolve this, `job_command` in config file should be of the form `"optenv/Scripts/python.exe"`.
This is actually stated in the user manual...
